content/docs/cml/self-hosted-runners.md

```diff
@@ -83,24 +83,27 @@ you do it your way :)
-| `--cloud-spot-price`  | Spot max price. If not specified it takes current spot bidding pricing.                                       | default: `-1`                                                                                                  |
-| `-h`                  | Show help                                                                                                     | Boolean                                                                                                        |
+| Argument                 | Description                                                                                                   | Values                                                                                                         |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Show version number                                                                                           | Boolean                                                                                                        |
+| `--labels`               | Comma delimited runner labels                                                                                 | Default: `cml`                                                                                                 |
+| `--idle-timeout `        | Time in seconds for the runner to be waiting for jobs before shutting down. 0 waits forever.                  | Default: `300`                                                                                                 |
+| `--name`                 | Name displayed in the repo once registered                                                                    | Default: `cml-` followed by a unique identifier, i.e. `cml-cfwj9rrari`                                         |
+| `--single`               | If specified, exit after running a single job.                                                                | Boolean                                                                                                        |
+| `--reuse`                | Don't launch new runners if there is, at least, another one with the same `--name` or containing `--labels`   | Boolean                                                                                                        |
+| `--driver`               | If not specified, driver is inferred from environmental variables                                             | Choices: `github`,`gitlab`                                                                                     |
```

Comment:
Is "type of driver" the correct way to say this? Or something like "Driver platform" perhaps?

Reply:
Please ignore this comment on --driver for now. Didn't mean to post this yet. Resolving!

Reply:
The _driver_ concept is a bit of a stretch and describes a platform-specific continuous integration runner [(like this)](https://github.com/actions/runner) that's wrapped by our code. I like the idea of "driver platform" even if the platform part becomes more insightful than the driver part.

Reply:
Thanks for the info. It will be useful for a separate review I'm preparing... ⌛